Base/usr/share/man/man7/Audio-subsystem.md

```diff
@@ -0,0 +1,60 @@
+
+LibDSP was started to support development efforts in Piano, but it is intended as a general-purpose audio processing library, building on the groundwork from LibAudio. Therefore, users of LibDSP must be familiar with LibAudio classes and concepts, as they are used extensively in LibDSP.
+
+### Applications
+
+This is a non-exhaustive list of applications that use audio. Most of these follow the good practices laid out in this manual page and may serve as a template for new audio applications.
+
+* **Piano** is a sequencer/tracker and synthesizer.
+* **aplay** is a command line audio file playback utility.
+* **SoundPlayer** is a UI audio file player with extra features such as playlist support and audio visualizations.
+* [**asctl**](../man1/asctl.md) is a command line audio server control utility.
+* **Applets/Audio** (AudioApplet) is a taskbar applet for setting audio parameters through a UI.
+
+### Volume
+
+Audio volume is more complicated than just multiplying a (digital or analog) audio signal with a percentage volume value. As the human hearing is logarithmic, volume changes also need to be logarithmic. An excellent article on the topic can be found [here](https://www.dr-lex.be/info-stuff/volumecontrols.html).
+
+For the SerenityOS audio system, the following applies: Userland applications and libraries that do their own volume changes need to be aware of the nature of volume. LibAudio provides utility functions for correctly handling volume, so these are to be used whenever applicable. For AudioServer, main and per-client volume is already handled correctly; to the outside, volume is linear between 0 and 1.
+
+For example: A program may set its client volume to 0.5 and the audio will be percieved as half as loud by a human. However, if the program wishes to change the volume beforehand, it needs to use logarithmic scaling, for example with LibAudio's built-in functionality.
+
+### Sample rate
+
+SerenityOS's audio system has no fixed sample rate. The sample rate is ultimately determined by the sound hardware and all parts of the system need to be aware of it. Audio samples passed to AudioServer are assumed to be at the system sample rate, meaning that audio applications need to query this sample rate and resample their audio data accordingly.
+
+Although the sample rate can change at any time, it is considered a rarely-changing value and most applications don't handle sample rate changes during operation. Therefore, when the system sample rate is changed, audio applications may need to be re-opened.
+
+## Files
+
+* [/dev/audio](../man4/audio.md)
+* AudioApplet and AudioServer have settings which are managed by ConfigServer.
```

Comment:
Can we point to the specific files where these settings are stored?

Reply:
No, if we want to change ConfigServer's way of storing them in the future, we would have to change this as well. That's a ConfigServer thing and it's better suited for its documentation.